permutohedral_lattice/CMakeLists.txt

```diff
@@ -2,9 +2,10 @@ cmake_minimum_required(VERSION 3.9)
 
 project(permutohedral_lattice LANGUAGES CXX CUDA)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CUDA_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CUDA_STANDARD 17)
 set(CUDA_SEPARABLE_COMPILATION ON)
+set(TENSORFLOW_FRAMEWORK "path_to_libtensorflow_framework.so")
```

Comment:
Is there a way to auto-detect where this file might be? If not, it's ok.

Reply:
Normally it does auto detect. But unfortunately, by default the libtensorflow_framework.so file is now replaced with libtensorflow_framework.so.2 file in the library folder. I was not able to find a way to make the cmake file detect libtensorflow_framework.so.2 file directly. The other way was to create to a symbolic link for which the user must have sufficient permissions or else they would not be able to do that.

Reply:
Wait so shouldn't it say path_to_libtensorflow_framework.so.2 then?

Reply:
Nope. We can only use .so file. The path in our case had the .so file in our permutohedral_lattice folder itself.